pkgs/tools/system/lanzaboote/tool.nix

```diff
@@ -0,0 +1,102 @@
+{ systemd
+, makeWrapper
+, stdenv
+, binutils-unwrapped
+, sbsigntool
+, makeRustPlatform
+, rustPlatform
+, rustc
+, cargo
+, fetchFromGitHub
+, lanzaboote-stub
+, lib
+, mkStdenvNoLibs
+, pkgsBuildTarget
+, buildPackages
+, overrideCC
+, writeShellScriptBin
+}:
+let
+  uefiPlatform = lib.systems.elaborate "${stdenv.hostPlatform.qemuArch}-windows" // {
+    rustc.config = "${stdenv.hostPlatform.qemuArch}-unknown-uefi";
+    useLLVM = true;
+  };
+  uefiLinker = writeShellScriptBin "lld-link" ''
+    exec ${buildPackages.llvmPackages_15.lld}/bin/lld-link "$@"
+    '';
+
+  uefiBintools = buildPackages.llvmPackages_15.clang.bintools.override {
```

Comment:
@amjoseph-nixpkgs Would that be okay if I wired this in the `cc-wrapper` directly to expose all the `lld-` variants if they are available for LLVM-based cc-wrappers so I would not need to do this here and everyone can benefit from it.